test/type.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+import assert from 'assert';
+import sinon from 'sinon';
+import typeValidators from '../src/type';
+
+describe('Type', function() {
+	beforeEach(function() {
+		console.error = sinon.stub();
```

Comment:
It'd be best to do `sinon.stub(console, 'error')`, as that way you'll be able to restore it on `afterEach` through `console.error.restore()`